scalalib/src/mill/javalib/android/AndroidAppModule.scala

```diff
@@ -124,33 +212,23 @@ trait AndroidAppModule extends JavaModule {
           androidSdkModule().androidJarPath().path.toString // Include Android framework classes
         )
     )
-
     PathRef(dexOutputDir)
   }
 
   /**
-   * Packages the DEX files and Android resources into an unsigned APK using the `aapt` tool.
+   * Packages DEX files and Android resources into an unsigned APK.
    *
-   * The `aapt` tool takes the DEX files (compiled code) and resources (such as layouts and assets),
-   * and packages them into an APK (Android Package) file. This APK file is unsigned and requires
-   * further processing to be distributed.
+   * @return A `PathRef` to the generated unsigned APK file (`app.unsigned.apk`).
    */
   def androidUnsignedApk: T[PathRef] = Task {
     val unsignedApk: os.Path = T.dest / "app.unsigned.apk"
-
-    os.call(
-      Seq(
-        androidSdkModule().aaptPath().path.toString,
-        "package",
-        "-f",
-        "-M",
-        androidManifest().path.toString, // Path to AndroidManifest.xml
-        "-I",
-        androidSdkModule().androidJarPath().path.toString, // Include Android JAR
-        "-F",
-        unsignedApk.toString // Output APK
-      ) ++ Seq(androidDex().path.toString) // Include DEX files
-    )
+    os.copy.over((T.dest / os.up / "androidResources.dest" / "res.apk"), unsignedApk)
```

Comment:
`os.copy` should be enough here, since the body of a `Task` gets wiped out every time before it runs (unless it has `persistent = true`